pkg/master/master_test.go

```diff
@@ -378,7 +378,7 @@ func TestValidOpenAPISpec(t *testing.T) {
 	}
 
 	// TODO(mehdy): The actual validation part of these tests are timing out on jerkin but passing locally. Enable it after debugging timeout issue.
-	disableValidation := true
+	disableValidation := false
```

Comment:
oh, the tests are passing. let me remove the comment and see if they are still passing.